core/src/main/java/org/apache/gravitino/storage/relational/utils/POConverters.java

```diff
@@ -440,21 +421,15 @@ public static TablePO initializeTablePOWithVersion(
    *
    * @param oldTablePO the old TablePO object
    * @param newTable the new TableEntity object
-   * @param needUpdateVersion whether need to update the version
    * @param newSchemaId the new schema id
    * @return TablePO object with updated version
    */
   public static TablePO updateTablePOWithVersionAndSchemaId(
-      TablePO oldTablePO, TableEntity newTable, boolean needUpdateVersion, Long newSchemaId) {
+      TablePO oldTablePO, TableEntity newTable, Long newSchemaId) {
     Long lastVersion;
     Long currentVersion;
-    if (needUpdateVersion) {
-      lastVersion = oldTablePO.getLastVersion() + 1;
-      currentVersion = lastVersion;
-    } else {
-      lastVersion = oldTablePO.getLastVersion();
-      currentVersion = oldTablePO.getCurrentVersion();
-    }
```

Comment:
if we only update the table name, should we also update the table version?

Reply:
The problem is that the data in `table_version_info` is tracked by the version, if we do not do this change, it will break the definition of the table.

Reply:
I know, so what's the problem? According to the design, any changes to a table will then try to delete the old version of the data in `table_version_info` and then insert a new version of the data.